src/Components/SignupLogin/Login.js

```diff
@@ -11,64 +11,44 @@ const Login = () => {
     setShowPassword((prevShowPassword) => !prevShowPassword);
 
   return (
-    <div>
```

Comment:
@kumarishalini6 Why have you edited this file and removed Navbar? Please make sure to not edit a file which is not concerned with the assigned issue.